### DISCUSSION TITLE
pp-ocr4 gpu训练内存飙升100%，调整batch_size_per_card和num_workers 均无效

### DISCUSSION THREAD

**Discussion by xiaoyang1227** [PaddlePaddle/PaddleOCR#12791] [Q&A]
pp-ocr3 gpu 训练正常

**Comment by ErshovVE** [comment]
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/75930878/8bf06167-94e5-4344-a43b-8911f50e3330)
I encountered a similar problem, try reducing the parameter first_bs
